src/app/cfg.rs

```diff
@@ -1,34 +1,43 @@
-use super::types::{SinkFn, UpdateFn, ViewFn, WindowEventsFn};
-use super::{builder::IntoAfterMount, MountType};
+use super::types::{UpdateFn, ViewFn};
 use crate::virtual_dom::IntoNodes;
-use std::marker::PhantomData;
 use std::rc::Rc;
 
 #[allow(clippy::module_name_repetitions)]
-pub struct AppInitCfg<Ms, Mdl, INodes, GMs, IAM: ?Sized>
+pub struct AppCfg<Ms, Mdl, INodes>
 where
     Ms: 'static,
     Mdl: 'static,
     INodes: IntoNodes<Ms>,
-    IAM: IntoAfterMount<Ms, Mdl, INodes, GMs>,
 {
-    pub mount_type: MountType,
-    pub into_after_mount: Box<IAM>,
-    pub phantom: PhantomData<(Ms, Mdl, INodes, GMs)>,
+    pub(crate) document: web_sys::Document,
+    // @TODO_B: get more context of use in order to understand how to remove Element mount point
+    mount_point: web_sys::Element,
```

Comment:
Not sure you can remove this, as the `mount_point` is necessary for displaying the VDOM.